.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -23,16 +23,23 @@
 - _BE GENEROUS to your reviewer and your future self!_
 
 ### Checks to be considered:
+If any of these don't apply to your situation, please write N/A (not applicable) next to them and tick them.
```

Comment:
Not sure about this, as then we're in the argument of why it's not applicable.
Under normal circumstances, why aren't all of these applicable?

Reply:
also, for a select few, we've got "if appropriate"... so perhaps one or the other.

Perhaps we can try without this initially, and identify where the contention is?

Reply:
good point, i guess most of the time these are all applicable. i guess in edge case repos (such as this PR in this repo), i've marked almost every single one as N/A because i'm suggesting changes to the PR template and so most of the points don't apply as far as i can tell.

worth considering i also found this in the [checklist plugin docs](https://github.com/stilliard/github-task-list-completed#skippable-tasks) - supposedly if a checklist item mentions N/A or OPTIONAL, it is skipped by the CI:
![image](https://github.com/SecclTech/.github/assets/143596892/83888530-a7bd-4d5b-a58c-d8b09a2cf2e4)

Reply:
updated it to say:
- if they write N/A on an item, the CI will skip over it (i tested this and it works)
- if they write N/A on an item, please write a justification as to why it doesn't apply

Reply:
but i will get rid altogether if you think i should